### PR TITLE
Add schedule, remove talks

### PR DESCRIPTION
**Pull request by cheeaun**
I assume we don't need the "Talk proposals" section anymore, replaced by this "Schedule" section. Please advise if we need it back. This could be merged once the schedule is finalized for production.

This change looks like this:

nov-02-2017 07-41-42

Notes:

- Spreadsheet is from http://gsx2json.com/api?id=1DsQpdJxF2N0zAR6UM1ornsI5p4dYTU3n6f3vAtQC1BU&sheet=1&columns=false
- All avatars now point to @choonkeat because spreadsheet will need to be updated with speaker avatar images.
- Schedule is fetched via the `gsx2json` service, with a `cache-first` strategy (get from `localStorage` and render first, while an `AJAX` call is done to fetch the latest one), to ensure schedule always load even when offline (or slow network) and the latest schedule will always load later if there are changes.
- The arrows and talk descriptions will appear conditionally based on the `talkdescription` and `speakername` keys.

### CODE REVIEW

File: themes/pacgeek/layouts/partials/schedule.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            ))
+          + '</td>'
+        + '</tr>';
+    });
+    html += '</table>';
+    document.getElementById('schedule-content').innerHTML = html;
+  };
+  function getSchedule(){
+    var key = 'talks-schedule';
+    var data = localStorage.getItem(key);
```
Comment (hongkheng) [inline themes/pacgeek/layouts/partials/schedule.html]: Could we add checks for `sessionStorage` when users are using private browsing mode in Safari?

**Comment by vinamelody** [comment]
Wait for @silverframe still finalising schedule and confirming speakers

**Review by hongkheng — CHANGES_REQUESTED** [review]
let hold the merge first

**Comment by elfgoh** [comment]
@silverframe has given the greenlight. Any other blockers?

**Comment by hongkheng** [comment]
ok let me remove the tag

**Review by hongkheng — APPROVED** [review]

**Pull request merged**